codewars16.py

```diff
@@ -3,16 +3,18 @@
 def arrange(s):
     if s:
         t = []
-        u = s.copy()
+        count = 0
         while len(t) < len(s):
-            if len(u) == 1:
-                t.append(u[0])
-                break
-            t.append(u[0])
-            t.append(u[-1])
-            u.reverse()
-            u.pop(0)
-            u.pop(-1)
+            if count % 2 == 0:
+                t.append(s[count])
+                if (count < (len(s) // 2)):
```

Comment:
Hi Piotr, great job and really nice improvement! Just a small tip for the future - there's no need to use the brackets for if statements. Your Python IDE most likely will even underline it to give you the same hint. See picture below :) 
![image](https://user-images.githubusercontent.com/35009392/179392727-0f849840-e9eb-4a13-b034-dcabfa433022.png)